test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -430,11 +446,12 @@ export default class GutenbergEditorComponent extends AbstractEditorComponent {
 		return blockClass.Expect( this.driver, blockID );
 	}
 
+	// @todo This and the `getTitle` method in this class look too similar. Refactor to a single method.
 	async titleShown() {
 		const titleLocator = By.css( '.editor-post-title__input' );
 		await driverHelper.waitUntilElementLocatedAndVisible( this.driver, titleLocator );
 		const element = await this.driver.findElement( titleLocator );
-		return await element.getAttribute( 'value' );
+		return await element.getProperty( 'textContent' );
```

Comment:
For these calls, we know we are working with the header block, which is now an `h1` with `content-editable` set to `true`. There's no fallback for inputs here, so this means we should only merge this PR once Gutenberg 10.4.0 is shipped to non-edge sites.